paths/paths_test.go

```diff
@@ -74,10 +82,3 @@ func TestConfig(t *testing.T) {
 		assert.Equal(t, tc.expectedPath, actual, tc.desc)
 	}
 }
-
-func TestXDGConfig(t *testing.T) {
```

Comment:
This test is useless now, as the behaviour of the `paths.Config()` only depends on `ConfigHome`, which was tested above.